manticore/core/memory.py

```diff
@@ -999,13 +994,14 @@ def read(self, address, size):
                             result[offset] = Operators.ITEBV(8, condition, Operators.ORD(value), result[offset])
             return map(Operators.CHR, result)
 
-    def write(self, address, value):
+    def write(self, address, value, force=False):
         '''
         Write a value at address.
         :param address: The address at which to write
         :type address: int or long or Expression
         :param value: Bytes to write
         :type value: str or list
+        :param force: Whether to ignore permissions
         '''
         size = len(value)
         if issymbolic(address):
```

Comment:
assert force is False in the symbolic address path?

Reply:
The reason i hesitated to add force to that path is because this is used to find cases of writing to unmapped memory, if we honor force, we can miss those. One way to side-step that is to only consider unmapped memory as crashing conditions, not just write failure.

Reply:
You are probably not using force=True on symbolic addresses. 
So I propose you just add the assert there.
assert force is False 
?

Reply:
Addressed in most recent commit